### DISCUSSION TITLE
create plan bug

### DISCUSSION THREAD

**Discussion by Validatrium** [subquery/subql#907] [Frontier TestNet - Bug Reporting]
link: https://testnet.thechaindata.com/plans/plans/default

if one reloads page and quickly presses "create plan" there will be Kusama and Polkadot in "Select specific deployment Id" field
but they will disappear in 1 sec (you may get lucky to press it and create specific plan)

**Comment by joesixpack** [comment]
Yes, the Deployment ID does not show up at all when purchasing a plan.  So no way to double check what you're actually purchasing before confirming.

**Comment by mzxyz** [comment]
@lilian79511 @CaiYiLiang
